README.md

```diff
@@ -136,21 +140,68 @@ See the [CONTRIBUTING](https://github.com/OpenTermsArchive/contrib-declarations/
 
 ### Installing
 
-This module is built with [Node](https://nodejs.org/en/). You will need to [install Node >= v14.x](https://nodejs.org/en/download/) to run it.
-
-Clone the repository and install dependencies:
+- This module is built with [Node](https://nodejs.org/en/). You will need to [install Node >= v14.x](https://nodejs.org/en/download/) to run it.
+- When refering to the root folder, it means the folder where you will be `git pull`ing everything.
+
+#### Declarations repository
+
+1. Locally clone your declaration repository, e.g., `git@github.com:OpenTermsArchive/contrib-declarations.git`.
+2. Go into your folder and initialize it, e.g., `cd contrib-declarations; npm install`.
+3. You can now modify your declarations in the `./declarations/` folder, following [these instructions](https://github.com/OpenTermsArchive/contrib-declarations/blob/main/CONTRIBUTING.md).
+4. When you want to test:
+    - If you want to test every declaration, run `npm test`.
+    - If you want to test a specific declaration, run `npm test <name of the service>`, e.g., `npm test HER`.
```

Comment:
We use `$`-prefixed syntax for variables in the rest of the README, let's keep this homogenous 🙂 
```suggestion
    - If you want to test a specific declaration, run `npm test $serviceId`, e.g., `npm test HER`.
```